pkg/transformers/transformer.go

```diff
@@ -82,6 +82,15 @@ const (
 	columnDynamicParam = "column"
 )
 
+type TransformerParameter struct {
```

Comment:
I think if we're introducing this new concept, we should rename the old one, otherwise we'll have `transformers.Parameter` and `transformers.TransformerParameter` types, which are ambiguous and confusing. 

I'd suggest using `Parameter` for this new one you're adding, and renaming the existing one to `ParameterValues`, since it aligns better with what it represents.